app/code/Magento/DirectoryGraphQl/etc/schema.graphqls

```diff
@@ -10,8 +10,8 @@ type Query {
 type Currency {
     base_currency_code: String
     base_currency_symbol: String
-    default_display_currecy_code: String
```

Comment:
It will Backward Incompatible Changes
We need to mark old fields as `deprecated` instead of removing them
thanks

Reply:
Fields were marked as deprecated